eclipse/updatesite-notp/pom.xml

```diff
@@ -71,7 +71,7 @@
 				</file>
 			</activation>
 			<properties>
-				<tools.mdsd.dependencies.eclipse-version>2020-12</tools.mdsd.dependencies.eclipse-version>
+				<tools.mdsd.dependencies.eclipse-version>2021-09</tools.mdsd.dependencies.eclipse-version>
 				<tools.mdsd.dependencies.ecoreworkflow-version>0.3.0</tools.mdsd.dependencies.ecoreworkflow-version>
```

Comment:
There is a mismatch between the version used in the target platform (0.2.0) and this version (0.3.0). I suggest to downgrade the version here because the target platform is already released with the told version.